content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -36,25 +36,21 @@ Enabling validation on your `rippled` server means providing a validator token i
 
 In a location **not** on your validator:
 
-1. Manually build and run the `validator-keys` tool, if you don't already have it installed via a `rippled` RPM. ***TODO: added this step for users who need to use the validator-keys tool delivered outside the rippled RPM. For example, they don't want to install rippled via RPM on their non-validator machine and just need to be able to generate the required keys and token. Okay?***
+1. Manually build and run the `validator-keys` tool, if you don't already have it installed via a `rippled` RPM.
 
     For information about manually building and running the `validator-keys` tool, see [validator-keys-tool](https://github.com/ripple/validator-keys-tool).
 
 2. Generate a validator key pair using the `create_keys` command.
 
-      - For an RPM-based `validator-keys` tool install, run:
+        $ validator-keys create_keys
 
-        ```
-        $ /opt/rippled/bin/validator-keys create_keys
-        ```
+      Sample output on Ubuntu:
 
-      - For a manually built `validator-keys` tool install, run:
+        Validator keys stored in /home/local/.ripple/validator-keys.json
```

Comment:
This path is probably still not correct. The [validator keys tool docs](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md#validator-keys) suggest it should be:

```
/home/ubuntu/.ripple/validator-keys.json
```

(In this case, `ubuntu` is presumably the username. Linux convention is `/home/{USER}/`.)

Reply:
:Wave: @mDuo13! Sorry for the misunderstanding here. Yes, I changed my username, which comes back in the actual output. to `local` - just to remove my name from the example. 😂  

In the validator keys tool doc - yes, they changed the username to `ubuntu`. I can change my example here to `ubuntu` -- would that work for you? 🙌 

We should probably come up with some style guide instructions for how to genericize these sample outputs/responses - what do you think?

Reply:
`.local` is actually a common file to be in one's home directory on Linux, so we should choose something different as a generic name. Preferably something no one could mistake as literal.

In some other places, we assume that there is a `rippled` user and the `rippled` service runs as that user after being installed by RPM (I think this is correct) but in this case, where we're instructing people to do something _not_ on their server, using a more generic name is appropriate.

Perhaps `my-user`?